src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderProgramProfileBuilder.kt

```diff
@@ -1,51 +1,23 @@
 package uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders
 
-import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.CourseActivity
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.OffenderBooking
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.OffenderProgramProfile
 import java.time.LocalDate
 
-@Component
-class OffenderProgramProfileBuilderFactory(
```

Comment:
All data creation is now done through the test data DSL so can get rid of the old builder factories